### PR TITLE
feat(editor): Use editor light theme if light mode

### LINKED ISSUE #984: day-mode should be extended to editor
**Which part of the project should be enhanced?**

Currently, when toggling between day and night mode, only the toolbar, and preview are changed.
I would like for the actual editor pane to be changed, too.

**Is your enhancement request related to a problem? Please describe.**

On a bright and sunny day, the dark editor is really really hard to use.

**Describe the solution you'd like**

The sun and moon toggle the editor from bright to dark, not just the toolbar

**Describe alternatives you've considered**

alternatively, if people find this offensive to their tastes, i can imagine having different sets of editor styles on the bottom to choose from.

Although i think this solution would be less intuitive and less accessible.

**Additional context**

I am "old" and I'm only getting older, and my eyes are only gonna get worse.

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Editor

### Description
This PR changes the editor theme to light if the app is in light mode

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### Related Issue(s)
Closes #984

### CODE REVIEW

File: src/components/editor-page/editor-pane/editor-pane.tsx [hunk 1]
```diff
@@ -154,7 +157,7 @@ export const EditorPane: React.FC<ScrollProps> = ({ scrollState, onScroll, onMak
         maxWidth={'100%'}
         basicSetup={true}
         className={codeMirrorClassName}
-        theme={oneDark}
+        theme={darkModeActivated ? oneDark : undefined}
```
Comment (DerMolly) [inline src/components/editor-page/editor-pane/editor-pane.tsx:160]: Didn't we already have a light editor theme? This look very bare bones.

**Review by DerMolly — CHANGES_REQUESTED** [review]

File: src/components/editor-page/editor-pane/editor-pane.tsx (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/editor-pane.tsx:160]: No one I know of. This is just the default light theme.

Comment (ErikMichelson) [reply src/components/editor-page/editor-pane/editor-pane.tsx:160]: Our previously writte code says this:

image

Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/editor-pane.tsx:160]: neat is no codemirror 6 theme

Comment (DerMolly) [reply src/components/editor-page/editor-pane/editor-pane.tsx:160]: Okay, so we need to find a good light theme for CM6. I added an issue https://github.com/hedgedoc/react-client/issues/2450

**Review by DerMolly — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Comment by mrdrogdrog** [comment]
There is still a bug that causes the linter fixes to not work if you switch the light/dark mode. I want to investigate this before merging

**Comment by mrdrogdrog** [comment]
The bug was been .. well not fixed but workarounded. The linter extension is recreated every time the dark/light mode is changed.

**Pull request merged**